packages/shared/src/types/category.ts

```diff
@@ -0,0 +1 @@
+export type Category = "server" | "client";
```

Comment:
I really don't like the term "Category" here. It's what we used on the Ecosystem page, so that is what I used here. Perhaps something like "IntendedOn", "IntendedFor", "RunsOn", "CompatibleWith", or something similar would work better.

@moredip, any recommendations?

Reply:
My vote is `Paradigm`!

Reply:
Do you think I should stick with the values of `server` and `client` if we go with `Paradigm`? In the spec, we use the term paradigm to describe static context vs dynamic context. I think the intent is more clear when you use the term `server` or `client` but I want to be careful mixing terminology.

@moredip @toddbaert @lukas-reining

Reply:
I really don't mind, but I think if I had to pick one, I'd pick `RunsOn` and `client/server`.

Reply:
I went with a combination of the two. The provider has an optional property `runsOn` which expects a `Paradigm`.